mflowgen/common/power-domains/outputs/params.tcl

```diff
@@ -0,0 +1,63 @@
+##### PARAMETERS #####
+# tile size/placement grid granularity
+# set x grid granularity to LCM of M3 and M5 and finfet pitches 
+# (layers where we placed vertical pins)
+set tile_x_grid 1.68
+# TODO: set pin_x_grid [lcm 0.07 0.08] 
+set pin_x_grid 0.56
+#set tile_x_grid [lcm $tile_x_grid 0.09]
+set tile_x_grid 5.04
+# set y grid granularity to LCM of M4 and M6 pitches 
+# (layers where we placed horizontal pins) and std_cell row height
+set tile_y_grid 2.88
+set pin_y_grid 0.08
+
+# PolyPitch grid parameters
+set polypitch_x 0.09
+set polypitch_y 0.576
```

Comment:
This can't be in the public repo... In fact, it seems like this script shouldn't be here at all. The design params aren't relevant to this flow, and the tech params can be acquired using `dbGet` commands

Reply:
Removed references to params.tcl. Buildkite status is same as before.